drizzlepac/haputils/svm_quality_analysis.py

```diff
@@ -248,18 +243,25 @@ def compare_num_sources(catalog_list, drizzle_list, json_timestamp=None, json_ti
     pnt_suffix = '_point-cat.ecsv'
     seg_suffix = '_segment-cat.ecsv'
 
+    # Get a list of the catalog files (*.ecsv) in the current working directory
+    catalog_files = []
+    basepath = os.getcwd()
+    for entry in os.listdir(basepath):
+        if os.path.isfile(os.path.join(basepath, entry)):
+            if entry.endswith("ecsv"):
+                catalog_files.append(entry)
+
```

Comment:
For future reference, all these lines could replaced with:

`catalog_files = glob.glob('*.ecsv')`